poll/public/handlebars/poll_studio.handlebars

```diff
@@ -2,21 +2,26 @@
 {{#each items}}
 <li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
     <div class="wrapper-comp-setting">
+        <div class="poll-move">
+            <button class="poll-move-up">&#9650;<span class="sr">&nbsp;move poll up</span></button>
+            <button class="poll-move-down">&#9660;<span class="sr">&nbsp;move poll down</span></button>
+        </div>
+        <button class="button action-button poll-delete-answer">Delete</button>
         <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
         <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
         {{#if image}}
         <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
-        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" /><br />
+        <label class="label setting-label" for="{{noun}}-img_alt-{{key}}">Image alternate text</label>
+        <input class="input setting-input" name="{{noun}}-img_alt-{{key}}" id="{{noun}}-img_alt-{{key}}" value="{{img_alt}}" type="text" /><br />
```

Comment:
@smarnach, sorry, I was just being just being polite.  I should have said more clearly that this was a requirement.  Since this could be client side validated, or at the very least, Studio side validated, I don't see how it would invalidate any previous polls.  Not making this required allows authors to create inaccessible content.  In this context, a label for a form control, a text equivalent is always required.  While we cannot guarantee that the author will add useful alt text, any alt text is better than none.  This must be mandatory.